app/src/main/java/kr/ac/kaist/team888/locator/Locator.java

```diff
@@ -69,53 +76,106 @@ public Locator(char letter) {
     return regions;
   }
 
-  private ArrayList<Stroke> transformClosedPath(ArrayList<Stroke> closedPath,
-                                                Region baseRegion, Region targetRegion) {
-    for (Stroke stroke : closedPath) {
-      stroke.setStartPoint(transformPoint(stroke.getStartPoint(), baseRegion, targetRegion));
-      stroke.setControlPoint(transformPoint(stroke.getControlPoint(), baseRegion, targetRegion));
-      stroke.setEndPoint(transformPoint(stroke.getEndPoint(), baseRegion, targetRegion));
-    }
+  /**
+   * Returns an array list of characters of the letter located by the locator.
+   *
+   * @return an array list of characters.
+   */
+  public ArrayList<HangulCharacter> getCharacters() {
+    return characters;
+  }
 
-    return closedPath;
+  /**
+   * Returns an array list of skeleton of the letter located by the locator.
+   *
+   * @return an array list of skeleton.
+   */
+  public ArrayList<ArrayList<Stroke>> getSkeletons() {
+    return skeletons;
   }
 
-  private Point2D transformPoint(Point2D point, Region baseRegion, Region targetRegion) {
-    Point2D baseMinPoint = baseRegion.getMinPoint();
-    Point2D baseDiffPoint = baseRegion.getMaxPoint().sub(baseMinPoint);
-    Point2D targetMinPoint = targetRegion.getMinPoint();
-    Point2D targetDiffPoint = targetRegion.getMaxPoint().sub(targetMinPoint);
+  /**
+   * Recalculate Paths and Circles respect to canvas region.
+   *
+   * <p>paths (from {@link Locator#getPaths()}),
+   * fixedCircles (from {@link Locator#getFixedCircles()}),
+   * controlCircles (from {@link Locator#getControlCircles()})
+   * are updated from this function call.
+   *
+   * @param canvasRegion Region of canvas
+   */
+  public void invalidate(Region canvasRegion) {
```

Comment:
It's telling that current data is invalid. "invalidate current data". I've meet convention from Android.Canvas.invalidate().